lms/envs/common.py

```diff
@@ -2268,6 +2268,7 @@
 STATIC_TEMPLATE_VIEW_DEFAULT_FILE_EXTENSION = 'html'
 
 SUPPORT_SITE_LINK = ''
+PASSWORD_RESET_SUPPORT_LINK = ''
```

Comment:
Nevermind -- learned relatively recently that this is only done in the edx-internal repository.